lbrynet/daemon/Daemon.py

```diff
@@ -2519,16 +2530,21 @@ def jsonrpc_claim_list_mine(self, account_id=None):
                     'amount': (float) amount assigned to the claim
                     'blocks_to_expiration': (int) number of blocks until it expires
                     'category': (str) "claim", "update" , or "support"
+                    'channel_name': (str) name of the channel prefixed with '@'
                     'claim_id': (str) claim ID of the claim
                     'confirmations': (int) number of blocks of confirmations for the claim
+                    'decoded_claim': (bool) whether or not the claim value was decoded
                     'expiration_height': (int) the block height which the claim will expire
                     'expired': (bool) true if expired, false otherwise
+                    'has_signature': (bool) included if decoded_claim
                     'height': (int) height of the block containing the claim
+                    'is_pending': (bool) *** Don't know what this means.  I either get "false" or null for the value. ***,
```

Comment:
This value should be True if the claim has not yet been included in a block

Reply:
@jackrobison see https://github.com/lbryio/lbry/issues/1470